core/new-gui/multi-target/.gitignore

```diff
@@ -0,0 +1,39 @@
+# See http://help.github.com/ignore-files/ for more about ignoring files.
+
+# compiled output
```

Comment:
Do we need another gitignore file? can we merge this with the root level gitignore?